x/ibc/02-client/handler.go

```diff
@@ -48,6 +55,7 @@ func HandleMsgCreateClient(ctx sdk.Context, k Keeper, msg exported.MsgCreateClie
 			EventTypeCreateClient,
 			sdk.NewAttribute(AttributeKeyClientID, msg.GetClientID()),
 			sdk.NewAttribute(AttributeKeyClientType, msg.GetClientType()),
```

Comment:
wanna update these other attribute keys to use `types.Attribute...`?

Reply:
I’ll remove the IBC alias in another PR 👍